docs/outputs.rst

```diff
@@ -142,7 +142,43 @@ sampled to those subject spaces.
 Confounds
 ---------
 
-See implementation on :mod:`~fmriprep.workflows.bold.confounds.init_bold_confs_wf`.
+The :abbr:`BOLD (blood-oxygen level dependent)` signal measured with fMRI is a mixture of fluctuations
+of both neuronal and non-neuronal origin. Neuronal signals are measured indirectly as changes
+in the local concentration of oxygenated hemoglobin. Non-neuronal fluctuations in fMRI data
+may appear as a result of head motion, scanner noise, or physiological fluctuations
+(related to cardiac or respiratory effects) (see Greve et al. [Greve2013]_ for detailed review of the possible
+sources of noise in fMRI signal).
+
+*Confounds* (or nuisance regressors) are variables representing potential fluctuations
+of non-neuronal origin. Such non-neuronal fluctuations may drive spurious results in fMRI data analysis,
+including standard activation :abbr:`GLM (General Linear Model` and functional connectivity analyses.
+It is possible to minimize confounding effects of non-neuronal signals by including them as nuisance regressors
+in the :abbr:`GLM (General Linear Model` design matrix or regressing them out from
+the fMRI data - a procedure known as *denoising*.
+There is currently no consensus on an optimal denoising strategy in the fMRI community.
+Rather, different strategies have been proposed, which achieve different levels of trade-off between
+how much of the non-neuronal fluctuations are effectively removed, and how much of neuronal fluctuations
+are damaged in the process. The fMRIprep pipeline generates a large array of possible confounds.
+
+The best known confounding variables in neuroimaging are the six head motion parameters
+(three rotations and three translations) - the common output of the head motion correction
+(also known as *realignment*) of popular fMRI preprocessing software
+such as `SPM <https://www.fil.ion.ucl.ac.uk/spm/software/spm12/>`_
+or `FSL <https://fsl.fmrib.ox.ac.uk/fsl/fslwiki>`_. One of the biggest advantages of fMRPrep
```

Comment:
FSL link is already in ``links.rst``, so it can be used by reference:
```suggestion
or FSL_.
One of the biggest advantages of *fMRIPrep*
```
Also: added newline, fixed style, and a typo in fMRIPrep.

Reply:
Done!